editor/plugins/org.fusesource.ide.projecttemplates/template-sources/simple/ose/simple-ose-log-springboot-fuse7.1/pom.xml

```diff
@@ -67,9 +66,9 @@
 				</configuration>
 			</plugin>
 			<plugin>
-				<groupId>org.springframework.boot</groupId>
+				<groupId>org.jboss.redhat-fuse</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
-				<version>${spring-boot.version}</version>
+				<version>${fuse.version}</version>
```

Comment:
Is that version issue no longer present?

Reply:
Can you precise which version issue exactly?

here we are switching from io.fabric8:spring-boot-maven-plugin to org.jboss.redhat-fuse:spring-boot-maven-plugin to allow use of the bom version for it and ensure the springboot 1.x or 2.x is correctly used depending on the bom version used.